src/coordinator.coffee

```diff
@@ -486,6 +486,17 @@ class Coordinator extends EventEmitter
           port: parts[3]
       graph.connections.push edge
 
+    iipIds = Object.keys(@iips).sort()
```

Comment:
This is similar to how fbp-graph does it: normal edges are exported first, then IIPs

https://github.com/flowbased/fbp-graph/blob/master/src/Graph.coffee#L774